.travis.yml

```diff
@@ -8,7 +8,7 @@ python:
   - '3.6'
 
 install:
-  - pip install --upgrade setuptools
+  - pip install --upgrade pip setuptools
   - pip install -r requirements-dev.txt
 
 script:
```

Comment:
You also need to set `PYPI_UPLOAD: "true"` env var in the deploy step for it to work.